roles/custom/matrix-bridge-zulip/defaults/main.yml

```diff
@@ -0,0 +1,125 @@
+# SPDX-FileCopyrightText: 2021 - 2022 Toni Spets
+# SPDX-FileCopyrightText: 2022 - 2023 Nikita Chernyi
+# SPDX-FileCopyrightText: 2022 - 2025 Slavi Pantaleev
+# SPDX-FileCopyrightText: 2022 MDAD project contributors
+# SPDX-FileCopyrightText: 2022 Marko Weltzer
+# SPDX-FileCopyrightText: 2023 Samuel Meenzen
+# SPDX-FileCopyrightText: 2024 - 2025 Suguru Hirahara
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+---
+# MatrixZulipBridge is a puppeting appservice bridge for Zulip
+# Project source code URL: https://github.com/GearKite/MatrixZulipBridge
+
+matrix_bridge_zulip_enabled: true
```

Comment:
`matrix_zulip_bridge_*` vars look more natural, IMO. Don't steam and go-skype bridge follow the same pattern?